INSTALL.rst

```diff
@@ -279,6 +281,9 @@ the creation of the virtual environment, configure cvc5 with ``./configure.sh --
 If configured with ``--pythonic-path=PATH``, the build system will expect the Pythonic API's source to be at ``PATH``.
 Otherwise, if configured with ``--auto-download``, the build system will download it.
 
+Installing the Python bindings after building from source requires a Python environment with
+pip version 20.3 or higher.
```

Comment:
```suggestion
pip version 23.0 or higher.
```

Reply:
I really meant 20.3. pip is currently used for two tasks: building the wheel and installing the wheel. The first task requires pip >= 23.0 and the second task requires pip >= 20.3. The Python environment and the version of pip for these two tasks can be different since the build system creates a virtual environment for building the wheel by default.

Reply:
Ah ok, looked like a mix-up :D